drivers/cache/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Do we need this file @carlocaione ?

Reply:
Yes. Otherwise CMake complain about a missing file.